ironfish/src/network/peerNetwork.ts

```diff
@@ -396,12 +398,14 @@ export class PeerNetwork {
    * Call close when shutting down the PeerNetwork to clean up
    * outstanding connections.
    */
-  stop(): void {
-    this.started = false
-    this.peerConnectionManager.stop()
-    this.peerManager.stop()
-    this.webSocketServer?.close()
-    this.updateIsReady()
+  async stop(): Promise<void> {
+    await Promise.allSettled([
+      (this.started = false),
+      this.peerConnectionManager.stop(),
+      await this.peerManager.stop(),
```

Comment:
Similar comment here where the `await` inside of a Promise.allSettled isn't needed. I think you could drop the `Promise.allSettled` for now, since as far as I know there's only one async function here right now.